### PR TITLE
Update sfb-server-updates.md

### PR DESCRIPTION
**Pull request by erwinbierens**
Added latest CU updates for SfB2015/2019 and Lync 2013 server.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/sfb-server-updates.md [hunk 1]
```diff
@@ -49,6 +50,7 @@ KB 4470124 is the master KB for Skype for Business Server 2019 which has all the
 
 |Package Name|KB Number|Release Date|
 |:--- |:--- |:--- |
+|Skype for Business Server 2015 Cumulative Update 12|[KB 3061064](https://support.microsoft.com/kb/3061064)| July, 2020 |
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/sfb-server-updates.md:53]:
```suggestion
|Skype for Business Server 2015 Cumulative Update 12|[KB 3061064](https://support.microsoft.com/kb/3061064)| July 2020 |
```

**Comment by scanum** [comment]
@Ariantor PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by tiburd** [comment]
Because of the age of this unmerged PR and the file conflicts, I'm closing this PR.

**Comment by tiburd** [comment]
cc: @scanum

**Pull request closed without merging**